### ISSUE TITLE
Thread argument is inconsistent between terminal and netscript

### ISSUE THREAD

**Issue by bupjae** [bitburner-official/bitburner-src#309]
Version: `Bitburner v2.2.1 (c46cedd5)`

Reproduce step:
1. Create `child.js`
```js
export async function main(ns) { ns.tprint(ns.args[0], ": ", ns.getRunningScript().threads); }
```

2. Create `parent.js`
```js
export async function main(ns) { ns.run('child.js', 1.9, 'netscript'); }
```

3. On terminal, type `run child.js -t 1.9 terminal; run parent.js`

Expected result:
```
child.js: terminal: X
child.js: netscript: X
```
X can be 1 or 2, as same value

Actual result:
```
child.js: terminal: 2
child.js: netscript: 1
```

**Comment by Snarling** [comment]
In either case, it should probably fail and tell you to provide an integer.

**Comment by Snarling** [comment]
Fixed by #366
